internal/supported_types/aws_supported_types.go

```diff
@@ -0,0 +1,18 @@
+package supported_types
+
+import (
+	"context"
+
+	"github.com/RHEnVision/provisioning-backend/internal/clients"
+)
+
+func FilterInstanceTypes(_ context.Context, it *[]clients.InstanceType) (*[]clients.InstanceType, error) {
```

Comment:
So why do we need the interface?

Reply:
Oh! I think I know what Lukas wanted... with the interface we would be able to generalize better, but I do not think it is necessary ATM?

Reply:
Well, I thought that the type will be shared, I mean, Azure has the very same parameters: name, vcpus, cores, memory... The same for GCE. Then it could be actually generic. Maybe I am wrong, I don't mind "AWS" in the function name it can be removed once we add Azure...

..which is like next Monday ;-)